Refactor: Buildscripts

emyfops opened the pull request:
This pull request changes few things in the buildscripts

- `project.properties` were changed to `property(...)`
- You can now include the dependencies jar files directly into the jar file using `includeMod` and `includeLib`

Natan515 commented:
I can't accept it yet sorry, they told me to wait till they update the settings.

l1ving commented:
It takes like 30 seconds to approve it in the dashboard, or remove the badge if you don't want to fix it.

Avanatiker commented:
Yea we work on the permission stuff :) thanks a lot! this was just a bridging solution

Natan515 commented:
> It takes like 30 seconds to approve it in the dashboard, or remove the badge if you don't want to fix it.

Yes but they asked me to wait till they fix perms. Even if it takes 1 sec

emyfops committed:
Refactor: Buildscripts

emyfops committed:
Fix: Absolute flatdir path

emyfops committed:
Refactor: Don't include the jars

emyfops committed:
Refactor: Misinput

It was a misinput, MISINPUT
CALM DOWN
YOU CALM THE F DOWN

emyfops committed:
Feature: 1.20.+ version

The minimum fabric loader version is based on the fabric kotlin library minimum fabric loader version

emyfops committed:
Fix: Mixin mod conflict

emyfops committed:
Feature: Forge 1.19.3 to 1.20.x

emyfops committed:
Removed maven name

emyfops committed:
Global val instead of absolute path

emyfops committed:
Fix: Forge dev KFF

emyfops committed:
Fix: misinput

it was misinput CALM DOWN

emyfops committed:
Fix: Crash on forge dev env

emyfops committed:
Feature: NeoForge modloader

emyfops committed:
Refactor: Change module name to com.lambda.common

emyfops committed:
Fix: Forge dev env

emyfops committed:
Revert "Refactor: Change module name to com.lambda.common"

This reverts commit af8c6a15866f85abcbd5f629d8eae2eae1a9cc18.

emyfops committed:
Removed comment

emyfops committed:
Merge branch 'master' into refactor/buildscript

emyfops committed:
Fix: Crash at launch

emyfops committed:
Fix: Fabric

emyfops committed:
Removed bundle

emyfops committed:
Fix: NeoForge

emyfops committed:
Avoid including mod jar in fabric

emyfops committed:
Removed log entrypoints

emyfops committed:
Fix: Forge

emyfops committed:
Fix: Missing mods forge env

Avanatiker committed:
Small changes

emyfops committed:
Fix: Forge dev and prod
